src/Components/HouseDemandContext.tsx

```diff
@@ -27,6 +27,10 @@ export type HouseDemand = {
   setHasGarden: React.Dispatch<React.SetStateAction<boolean>>;
   hasCFO: boolean;
   setHasCFO: React.Dispatch<React.SetStateAction<boolean>>;
+  hasFirstWaitress : boolean;
```

Comment:
First waitress is a milestone so should be included in that object

Reply:
Good point, it's resolved.